src/main/java/duke/Event.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+public class Event extends Task{
+    /** Start time of event */
+    protected String from;
+    /** End time of event */
+    protected String to;
```

Comment:
Perhaps you can use a more intuitive noun as the variable name, like start or startTime and end or endTime?